<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-8AHTIUM" ignored="0" name=" loan" tests="5" failures="1" timestamp="16 Aug 2018 17:56:55 GMT" time="0.143" errors="0">
  <testcase name="WebsiteCarLoan" time="0.016" classname="test.parameters"/>
  <testcase name="users" time="0.007" classname="test.parameters"/>
  <testcase name="users" time="0.018" classname="test.parameters"/>
  <testcase name="users" time="0.002" classname="test.parameters"/>
  <testcase name="users1" time="0.035" classname="test.parameters">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.parameters.users1(parameters.java:46)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- users1 -->
</testsuite> <!--  loan -->
